### PR TITLE
Feature/add activity

### PR DESCRIPTION
**Pull request by magnify**

### CODE REVIEW

File: web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.fieldset {
+  border: 0;
+  magin: 0;
```
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss]: magin?

File: web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/radio.twig [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <label class="radio">
-  <div class="radio--label">{{ radio.label }}</div>
+    <div class="radio--label">{{ radio.label }}</div>
```
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/radio.twig]: Wierd space?

File: web/themes/custom/genlyd_aarhus/source/_patterns/01-modules/section-text-with-icon.twig [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{% embed "@modules/grid/grid-item-header-section.twig" %}
+  {% block content %}
+    {% embed "@modules/text-with-icon.twig" with { textWithIcon: textWithIcon } %}
+      {% block icon %}
+        {% include iconName ? "@elements/icons/icon-" ~ iconName ~ ".twig" : "@elements/icons/icon-arrow-dark-green-small.twig" %}
```
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/01-modules/section-text-with-icon.twig:5]: Perhaps default should be en empty icon?

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig [hunk 4]
```diff
@@ -0,0 +1,626 @@
...
+
+      {% embed "@modules/grid/grid-item.twig" %}
+        {% block content %}
+
+          {{ form.actions.submit }}
+          {{ form.actions.publish }}
+
+        {% endblock %}
+      {% endembed %}
+
```
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig:619]: Buttons in pattern-lab?

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig [hunk 5]
```diff
@@ -0,0 +1,626 @@
...
+    {% endblock %}
+  {% endembed %}
+
+  {{ form.form_build_id }}
```
Comment (tuj) [inline web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig]: {% if form %}

**Review by tuj — CHANGES_REQUESTED** [review]

File: web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss (hunk 1 above)
Comment (magnify) [reply web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss]: Whoa!

Comment (magnify) [reply web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/fieldset.scss]: magin is the new margin.

File: web/themes/custom/genlyd_aarhus/source/_patterns/01-modules/section-text-with-icon.twig (hunk 3 above)
Comment (magnify) [reply web/themes/custom/genlyd_aarhus/source/_patterns/01-modules/section-text-with-icon.twig:5]: Let's stick with this for now. We will probably get back to it.

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig (hunk 4 above)
Comment (magnify) [reply web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig:619]: Waiting for merge with 5 step form code, after merge I will take a look at getting it to work in Pattern Lab.

**Comment by magnify** [comment]
Commit changes, please review.

File: web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig (hunk 4 above)
Comment (tuj) [reply web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig:619]: Ok

**Pull request merged**